### PR TITLE
Add front.jpg to the list of valid album art images.

### PR DESCRIPTION
**Pull request by randrey**
**Changes**
Allows front.jpg to be used as album art when cover/folder image is not available.

### CODE REVIEW

File: MediaBrowser.LocalMetadata/Images/LocalImageProvider.cs [hunk 1]
```diff
@@ -218,7 +218,8 @@ private void PopulateImages(BaseItem item, List<LocalImageInfo> images, List<Fil
             "folder",
             "poster",
             "cover",
-            "default"
+            "front",
+            "default",
```
Comment (dkanada) [inline MediaBrowser.LocalMetadata/Images/LocalImageProvider.cs:222]:
```suggestion
            "default"
```

Comment (randrey) [reply MediaBrowser.LocalMetadata/Images/LocalImageProvider.cs:222]: It's actually recommended to have the coma - https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1413.md
Do you want it removed anyway?

Comment (dkanada) [reply MediaBrowser.LocalMetadata/Images/LocalImageProvider.cs:222]: That would be a question for @Bond-009, what do you think?

Comment (Bond-009) [reply MediaBrowser.LocalMetadata/Images/LocalImageProvider.cs:222]: > It's actually recommended to have the coma - https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1413.md
Do you want it removed anyway?

It isn't https://github.com/jellyfin/jellyfin/blob/master/jellyfin.ruleset#L26

**Comment by randrey** [comment]
Closing as I found a better way of achieving desired effect without making changes to the core code.

**Pull request closed without merging**